backend/db/utils.py

```diff
@@ -276,6 +276,31 @@ def is_table_up_to_date(table_name: str, skip_if_updated_within_hours: int = Non
     return check_if_updated(last_updated_key, skip_if_updated_within_hours)
 
 
+def reset_refresh_status(local=False):
```

Comment:
## Add: `reset_refresh_status()`
Meant to only be run manually during development.

This resets the state of all variables in `manage` related to whether or not the DB is currently active. It was starting to become a pain to have to do this manually because I just started to introduce more such variables for the derived tables.